src/pages/resources/reference/push-notifications/templates/tabs/rating.md

```diff
@@ -0,0 +1,96 @@
+| Rating icon in selected state | ✅ | `adb_rate_selected_icon` | string | URI or name for a rating icon to use when selected in the notification. If the URI is valid, the remote image is downloaded and cached on the device to be used as the rating icon. If a name is provided, the app's drawable resources are checked for an image file with the same name. If no valid URI or name is provided then no notification is constructed. |
+| Rating icon action | ✅ | `adb_rate_act` | string | **The value is an encoded JSON string.**<br />Three to five values to be used for rating confirmation, each containing a a click-through URI and type.<br />When decoded, the string contains an array of the following objects:<br /><ul><li>`uri` (required) - URI to be handled when the user selects this rating and clicks confirm</li><li>`type` (required) Type of link represented in `uri` - one of "WEBURL", "DEEPLINK", or "OPENAPP".</li></ul> |
+
+## Example
+
+<img src="./../assets/rating.gif" width="50%" height="50%"/>
+
+Below is a sample of what a payload might look like for a product rating notification:
+
+```json
+{
+  "message": {
+    "token": "FCM_TOKEN",
+    "android": {
+      "collapse_key": "new message"
+    },
+    "data": {     
+      "adb_version": "1",
+      "adb_title": "Thanks for shopping with us!",
+      "adb_body": "Please take a moment to rate your recent purchase.",
+      "adb_sound": "chaChing",
+      "adb_image": "https://sneakerland.com/products/shoe1/shoe.png",
+      "adb_uri": "https://sneakerland.com/products/rating/shoe1",
+      "adb_a_type": "WEBURL",
+      "adb_icon": "ic_shoe",
+      "adb_n_count": "1",
+      "adb_n_priority": "PRIORITY_LOW",
+      "adb_template_type": "rate",
+      "adb_body_ex": "We hope you are enjoying your new shoes. Please click on a rating below to help us cater to your future footware needs!",
+      "adb_clr_body": "00EE00",
+      "adb_clr_title": "AABBCC",
+      "adb_clr_icon": "123456",
+      "adb_clr_bg": "000000",
+      "adb_rate_unselected_icon":"https://cdn-icons-png.freepik.com/256/1077/1077035.png?semt=ais_hybrid",
+      "adb_rate_selected_icon":"https://cdn-icons-png.freepik.com/256/833/833472.png?semt=ais_hybrid",
+      "adb_rate_act": "[{\"uri\":\"https://www.adobe.com\", \"type\":\"WEBURL\"},{\"type\":\"OPENAPP\"},{\"type\":\"DISMISS\"},{\"uri\": \"https://www.adobe.com\", \"type\":\"WEBURL\"},{\"uri\":\"instabiz://opensecond\", \"type\":\"DEEPLINK\"}]"
+    }
+  }
+}
+```
+
+## Usage recommendations
+
+The tables below contain guidelines for your push notification content. **These recommendations help your text and images display reliably across multiple devices.** These values are guidelines only - it is still recommended to test a notification prior to sending it.
```

Comment:
Similar comment for the it is recommended as above: "... guidelines only - you should still test a notification..."